Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -1429,6 +1452,69 @@ - (void)didIgnoreFulfillmentWithNoAuthorizationPresent
 
 #endif
 
+- (void)fulfillAxisNowLicense:(NSURL *)fileUrl
```

Comment:
it would be great if this could live outside of `NYPLMyBooksDownloadCenter`. NYPLMyBooksDownloadCenter is a huge obnoxious class that just keeps growing. To be fair, I don't know how feasible that is, and I don't want to derail you into a refactor, but I wanted to point it out. 

I also wonder if we can avoid having `NYPLMyBooksDownloadCenter` be the `AxisNowDelegate`. And again, the same feasibility disclaimer applies of course.

Reply:
Agreed. I am of the opinion that every new feature should act like a plug and play device. Getting it entangled in previously written code is a really bad idea.